frontend/components/forms/searchable_user_list.jsx

```diff
@@ -12,42 +12,51 @@ class SearchableUserList extends React.Component {
   }
 
   componentDidMount(){
-    if (this.props.modalType === 'EDIT_CHANNEL_MODAL') {
-      this.props.selectUsers(this.props.prevUserIds);
+    const { modalType, selectUsers, prevUserIds } = this.props;
+    if (modalType === 'EDIT_CHANNEL_MODAL') {
+      selectUsers(prevUserIds);
     }
   }
 
   updateSearch(e){
     this.setState({query: e.target.value});
   }
 
-  render(){
+  createMatchEls(){
     const { users,
             selectedUsers,
             currentUserId,
-            selectUser,
-            deselectUser } = this.props;
-
+            selectUser } = this.props;
     const matches = findMatchingUsers(users, this.state.query);
-    const nonselectedMatches = matches.filter(user => {
+
+    const nonSelectedMatches = matches.filter(user => {
       return (!Object.values(selectedUsers).includes(user.id) &&
               user.id !== currentUserId);
     });
 
-    const matchEls = nonselectedMatches.map(
+    const matchEls = nonSelectedMatches.map(
        user => <SearchableUserListItem
                   key={user.id}
                   user={user}
                   selectUser={selectUser} />
     );
+    return matchEls;
```

Comment:
this callback would be cleaner if we opened parentheses, and on the next line we had the SearchableUserListItem component